### ISSUE TITLE
Many issues.

### ISSUE THREAD

**Issue by TukuToi** [zionbuilder/zionbuilder#553]
1. Upon install, I cannot create (edit) any component.
Whenever I try to edit a header, body or footer component in Theme Builder > Components (wp-admin/admin.php?page=zionbuilder-theme-builder), the Console throws `Unhandled Promise Rejection: TypeError: null is not an object (evaluating 'o.value.conditions')` and the editor is nothing, just a WPOD, see
<img width="1792" alt="Screenshot 2021-06-28 at 21 10 49" src="https://user-images.githubusercontent.com/11800236/123651021-554f1d00-d855-11eb-86a9-9fa2a3108d7b.png">

2. It is advertised that we can select templates for header, footer, and content, even on archives I believe I have read.
But I can't figure out where, and how. As far I can see the only place is as described in 1) above, and that doesn't work since we can't edit those components at all du to WPOD and js error.

3. When I create a Template in Zion Builder > Zion Builder > Templates, I can add and edit and save things, but I can't assign that template anywhere. I can insert it with shortcode, but that is pointless, as it will simply be used where the ShortCode is inserted, thus, this is far from "Theme Building"

Bottom line, it does not seem to work for me.

This was tested with PRO version on a 7.x PHP install with huge amounts of server resources so I don't think it is a server problem.
No plugins where used but Zion, and Theme 2021 and 2019 where utilised.

Any feedback on this?

Thanks

**Comment by zauan** [comment]
Hi there,

the mentioned error was resolved in v2.7.1.

Regarding theme builder, please check the following resources on how to use it:
https://www.youtube.com/watch?v=S-GvgnOLGpg&t=6s
https://www.youtube.com/watch?v=HW88QKMidjg&t=299s

If you need further help, please don't hesitate to contact us at https://zionbuilder.io/contact/ or open an additional ticket
